tests/store/metadata/oxigraph/test_oxigraph_topic.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pytest
+
+from pydantic import ValidationError
+
+from store.metadata.oxigraph.store import OxigraphMetadataStore
+import schemas
+
+
+def test_oxigraph_get_topic_with_subtopic_returns_valid_structure():
+    """
+    Confirm that the OxigraphMetadataStore.get_topic()
+    function returns a topic with sub_topic that matches the topic
+    schema.
+    """
+
+    os.environ["GRAPH_DB_URL"] = "http://localhost:7878"
```

Comment:
so for this, you should be able to use a pytest monkeypatch and conftest.py to set this envionrment variable but only for the length of the test run (at the moment this works, but you're overwriting the users env var, they might have it set to something else for a reasons).

there's a comment down a bit [here](https://stackoverflow.com/questions/46733332/how-to-monkeypatch-the-environment-using-pytest-in-conftest-py) that on the surface looks about right, though you might need to play. I'd start with the "Direct use of instance:" bit and see if it works.